core/kazoo_number_manager/src/knm_port_request.erl

```diff
@@ -332,6 +332,19 @@ transition_numbers(PortReq) ->
             {'error', PortReq}
     end.
 
+-spec create_and_activate(ne_binaries(), knm_number_options:options()) -> knm_numbers:knm_numbers_return().
+create_and_activate(Numbers, Options) ->
+    CreationResults = knm_numbers:create(Numbers, Options),
+    {_OK, Errored} = lists:partition(fun was_number_operation_successul/1, CreationResults),
+    case Errored of
+        [] ->
+            lager:debug("all numbers created, activating numbers"),
+            knm_numbers:move(Numbers, knm_number_options:assign_to(Options), Options);
```

Comment:
This is something that should be achieved by knm_numbers:create's assign_to already. The bug is elsewhere *X-Files theme*.

Since ticket was created multiple versions of this file ago, I'm curious of what crash caused this.